examples/local/README.md

```diff
@@ -0,0 +1,91 @@
+# Running cert-operator Locally
+
+**Note:** This should be used only for testing end development. See the
```

Comment:
I think this reads better. "This should only be used for testing and development."